could not determine kind of name for C.GtkPlug and C.GtkSocket

Hi,

I got an error when i try to build my application using gotk3

gtk version:
```
# pacman -Qi mingw-w64-x86_64-gtk3
Name            : mingw-w64-x86_64-gtk3
Version         : 3.24.13-1
```

Gotk3 version: master as of 9d4abcb1602160e43cba88eb9fa81beb4318ee53
```
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:105:26: could not determine kind of name for C.GtkPlug
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:37:28: could not determine kind of name for C.GtkSocket
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:73:34: could not determine kind of name for C.Window
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:160:2: could not determine kind of name for C.gtk_plug_construct
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:168:2: could not determine kind of name for C.gtk_plug_construct_for_display
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:182:7: could not determine kind of name for C.gtk_plug_get_embedded
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:175:7: could not determine kind of name for C.gtk_plug_get_id
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:189:7: could not determine kind of name for C.gtk_plug_get_socket_window
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:18:14: could not determine kind of name for C.gtk_plug_get_type
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:138:7: could not determine kind of name for C.gtk_plug_new
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:149:7: could not determine kind of name for C.gtk_plug_new_for_display
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:73:2: could not determine kind of name for C.gtk_socket_add_id
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:81:7: could not determine kind of name for C.gtk_socket_get_id
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:88:7: could not determine kind of name for C.gtk_socket_get_plug_window
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:17:14: could not determine kind of name for C.gtk_socket_get_type
vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:58:7: could not determine kind of name for C.gtk_socket_new
cgo: 
gcc errors for preamble:
In file included from vendor\github.com\gotk3\gotk3\gtk\socket_plug.go:7:
./socket_plug.go.h:2:8: error: unknown type name 'GtkSocket'
    2 | static GtkSocket *
      |        ^~~~~~~~~
./socket_plug.go.h:8:8: error: unknown type name 'GtkPlug'
    8 | static GtkPlug *
      |        ^~~~~~~
```

I think that it means my gtk does not have a `C.GtkSocket` nor `C.GtkPlug` but i'm using the last available gtk version (3_24).
Do you know how to fix it?

Thanks you.

--- a/gtk/socket_plug.go
+++ b/gtk/socket_plug.go
@@ -1,3 +1,5 @@
+// +build linux
+
 package gtk
 
 // #include <gtk/gtkx.h>

